website/docs/pipelines/spec/v1alpha1/pipeline.mdx

```diff
@@ -15,7 +15,7 @@ apiVersion: pipelines.weave.works/v1alpha1
 kind: Pipeline
```

Comment:
is there automation in place to update this spec when it changes in the codebase?

Reply:
I'm afraid not.